docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -77,25 +67,25 @@ openstack overcloud node provision \
 ----
 +
 When the update is complete, ensure that the {CephCluster} nodes have a
-new interface created and tagged with the appropriate VLAN associated with
+new interface created and tagged with the VLAN that is associated with
 `StorageNFS`.
 
 . Identify an IP address from the `StorageNFS` network to use as the Virtual IP
-address for the Ceph NFS service. This IP address must be provided in place of
-the `{{ VIP }}` in the example below. You can query used IP addresses with:
+address for the Ceph NFS service. You must provide this IP address in place of
+the `{{ VIP }}` in the example below:
 +
```

Comment:
I don’t see { VIP}. I see “Fixed IP Addresses”.

Reply:
yeah, this is Because the VIP is retrieved by the `openstack port list ..` command, that filters the output on the "Fixed IP Addresses" column, where we can find the VIP assigned by TripleO to the StorageNFS network.

Reply:
Yes, that makes sense.